README.md

```diff
@@ -1 +1,228 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [📷 Video Recording](#video-record)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Book Store <a name="about-project"></a>
+
+Our Booking app sends you to the real world of Books.[Hello Rails Back End App](https://github.com/NurkaAmre/Final-Capstone-Back-End).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>View Template Engine ".JS"</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="http://localhost:3000">React</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **On the main page user:** On the main oage user can see Books list in the library .
+- **User can se the details:** When the use r clicks on the book he can see the details of a Book.
+- **User can reserve a book:** In the details page, the user can click the "Reserve" button, for book reservation.
+- **User can delete the reservation:** When the user clicks the "Delete" button, the selected item is marked as removed and does not show on the main list anymore.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://nurka.onrender.com)
+
+- [Link to kanban](https://github.com/NurkaAmre/Final-Capstone-Back-End/projects/1)
+
+- [Link holding a picture after setting up kanban](https://github.com/NurkaAmre/Final-Capstone-Back-End/issues/13)
+
+Hello there we are a Team of 4 people wokring on this app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Code editor such as `VsCode`.
+- A web browser.
+- you need to have  `node` installed in your computer 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:NurkaAmre/Final-Capstone-Front-End.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Final-Capstone-Front-End
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/
+```
```

Comment:
- [ ] This command is meant for the backend project.  Since this is the frontend project, kindly remove it. 👍

Reply:
Omitted accordingly.